### DISCUSSION TITLE
Emails consistently landing in spam folders

### DISCUSSION THREAD

**Discussion by ripon202310** [postalserver/postal#3125] [Help with using Postal]
Hello Everyone,
I'm new to Postal and am facing challenges with my emails consistently landing in spam folders, despite adding the required DNS records. I've attached screenshots for your reference.I've followed the instructions and believe I've configured everything correctly. However, my emails are still not reaching the inbox.I'd appreciate your expert guidance on resolving this issue. Thank you

![terminal](https://github.com/user-attachments/assets/03d4dfc8-f22a-4dba-9352-b0d32328c40d)
![prt record](https://github.com/user-attachments/assets/15e3e6ba-451e-4d98-be0e-78d0ed936350)
![records in postal](https://github.com/user-attachments/assets/32768e83-dca0-4a1a-8efe-7592ffc71130)
![cloudflare records](https://github.com/user-attachments/assets/791f0639-1eb6-49ca-ac8a-ea27898bf68f)
![mail in spam folder](https://github.com/user-attachments/assets/848283c5-3bac-45a0-a9b5-764bf3f8200b)

**Comment by CooperDev-Code** [comment]
SPF is wrong, you have included spf.mail.adpox.xyz and that don't exist.

![image](https://github.com/user-attachments/assets/17855e9a-3191-4621-b290-94ec4caede18)


Feel free to reach out to me for help getting it going.

Regards
Mason

**Comment by CooperDev-Code** [comment]
Your missing a lot off DNS recoreds that need to be set up for the server itself.

add a new txt record.

spf.mail. 
v=spf1 ip4:45.10.154.120

add a new A record 
rp.mail
45.10.154.120

add a new MX
rp.mail   
10 mail.adpox.xyz

add a TXT record
rp.mail
v=spf1 a mx include:spf.mail.adpox.xyz ~all

You then need to get the server default DKIM 
log into the server cli and run postal default-dkim-record

Then add another txt record 
postal._domainkey.rp.mail
with the content off the above cmd 

Try that and see how you go.
